server/jwt.go

```
@@ -0,0 +1,79 @@
+	sess.Save(c.Request(), c.Response())
+}
+
+func removeUserSession(c echo.Context) {
+	sess, _ := session.Get("session", c)
+	sess.Options = &sessions.Options{
+		Path:     "/",
+		MaxAge:   0,
+		HttpOnly: true,
+	}
+	sess.Values[userIdContextKey] = nil
+	sess.Save(c.Request(), c.Response())
```

golangci-lint (1.64.8)

Error return value of `sess.Save` is not checked

(errcheck)